client/components/LearnerViewDirective.js

```diff
@@ -773,7 +800,15 @@ tie.directive('learnerView', [function() {
             for (var i = 0; i < feedbackParagraphs.length; i++) {
               if (feedbackParagraphs[i].isSyntaxErrorParagraph()) {
                 hasSyntaxError = true;
+                clearHighlight();
+                var errorContentArray = feedbackParagraphs[i].getContent()
+                  .split(' ');
+                var syntaxErrorLineNumber = errorContentArray[errorContentArray
+                  .length - 1];
+                highlightLine(parseInt(syntaxErrorLineNumber, 10));
                 break;
+              } else {
+                clearHighlight();
```

Comment:
Instead of using clearHighlight() in both the if and else case, why not just clearHighlight() once before the for loop?